### ISSUE TITLE
Just a variant

### ISSUE THREAD

**Issue by polluks** [codespell-project/codespell#2860]
https://github.com/codespell-project/codespell/blob/0f5c8fd59545fe2423dc03514e19cedad7623b8c/codespell_lib/data/dictionary.txt#L22488
Unicode uses the (Modern Greek-based) spelling "lamda" in character names, instead of "lambda", due to "the pre-existing names in ISO 8859-7, as well as preferences expressed by the Greek National Body".
https://en.wikipedia.org/wiki/Lambda#Character_encodings

**Comment by DimitriPapadopoulos** [comment]
Indeed, it's usually written λάμδα  and seldom λάμβδα. Yet, I don't know if it should be moved to the _rare_ dicrionary or just discarded.

**Comment by polluks** [comment]
I prefer _rare_ because it's rare.

**Comment by DimitriPapadopoulos** [comment]
So, what do you recommend? Moving to _rare_, but I wonder whether it should be discarded.

To the best of my knowledge, λάμδα has always been more common than λάμβδα in Greek. But the two consonants **μδ** are hard to pronounce and can naturally evolve into something like **μβδ** when pronounced. Anyway, the alternative “lambda” has been kept in English. However, “lamda” has prevailed in the Unicode Greek and Coptic scripts, as well as mathematical alphanumeric symbols.

**Comment by polluks** [comment]
I prefer _rare_, thanks anyway.

**Comment by DimitriPapadopoulos** [comment]
@polluks Please feel free to make a pull request as long as you have time.
